VotingApplication/VotingApplication.Web/Scripts/Controllers/BasicVoteController.js

```diff
@@ -53,6 +61,15 @@
 
         }
 
+        function addOption() {
```

Comment:
The next set of changes I'm making are looking at refactoring as much as possible out of the *VoteControllers and into the VotingPageController.